src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -123,6 +164,16 @@ public OffsetDateTime offsetDateTime(LocalDateTime localTime) {
      * или сообщение "dateTime can't be formatted!"
      */
     public String formatDate(LocalDateTime dateTime) {
-        return "Date can't be formatted!";
+        return dateTime.format(FORMATTER3);
```

Comment:
you never return  message dateTime can't be formatted!, please complete this task according to description

Reply:
Done